### DISCUSSION TITLE
PDTM installed in WSL is not working

### DISCUSSION THREAD

**Discussion by AliYar-Khan** [projectdiscovery/pdtm#300] [Q&A] [answered]
### Current Behavior:
pdtm in bash says command not found

### Expected Behavior:
should show options

**Comment by 5kyf1** [comment, marked as answer]
You can try to add Go path to bashrc,  eg: `export PATH=$PATH:$GOPATH/bin`

but most of the time the tool will auto generate the path, 
eg: `export PATH=$PATH:/home/kali/.pdtm/go/bin`
